scripts/adi_env.tcl

```diff
@@ -34,6 +34,21 @@ if {![info exists REQUIRED_QUARTUS_VERSION]} {
   set REQUIRED_QUARTUS_VERSION "23.2.0"
 }
 
+# Define the supported tool version
+set required_radiant_version "2023.2"
+if {[info exists ::env(REQUIRED_RADIANT_VERSION)]} {
+  set required_radiant_version $::env(REQUIRED_RADIANT_VERSION)
+} elseif {[info exists REQUIRED_RADIANT_VERSION]} {
+  set required_radiant_version $REQUIRED_RADIANT_VERSION
+}
+# Define the supported tool version
+set required_propel_builder_version "2023.2"
+if {[info exists ::env(REQUIRED_PROPEL_BUILDER_VERSION)]} {
+  set required_propel_builder_version $::env(REQUIRED_PROPEL_BUILDER_VERSION)
+} elseif {[info exists REQUIRED_PROPEL_BUILDER_VERSION]} {
+  set required_propel_builder_version $REQUIRED_PROPEL_BUILDER_VERSION
+}
+
```

Comment:
Should these be a single instruction and matched together in a single  if ?

Reply:
Yes, I think it's a good idea.

Reply:
You mean like this?
```suggestion
set required_propel_builder_version "2023.2"
set required_radiant_version "2023.2"
if {[info exists ::env(REQUIRED_LATTICE_VERSION)]} {
  set required_propel_builder_version $::env(REQUIRED_LATTICE_VERSION)
  set required_radiant_version $::env(REQUIRED_LATTICE_VERSION)
} elseif {[info exists REQUIRED_LATTICE_VERSION]} {
  set required_propel_builder_version $REQUIRED_LATTICE_VERSION
  set required_radiant_version $REQUIRED_LATTICE_VERSION
}

```